packages/stories/story-loader/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@republik/story-loader",
+  "version": "0.0.1",
+  "description": "renders a story component based on its name",
+  "private": true,
+  "main": "dist/index.js",
+  "module": "dist/index.mjs",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "node_modules/.bin/tsup src/index.ts --format cjs,esm --dts --minify --clean --out-dir dist",
+    "dev": "node_modules/.bin/tsup src/index.ts --format cjs,esm --dts --minify --clean --out-dir dist --sourcemap --watch"
+  },
+  "dependencies": {
+    "@republik/stories-example-pink": "*",
+    "@republik/stories-example-blue": "*",
+    "@republik/stories-the-dot-banks": "*"
+  },
+  "devDependencies": {
+    "@republik/tsconfig": "*",
+    "@swc/core": "^1.3.57",
+    "identity-obj-proxy": "^3.0.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "tsup": "^6.7.0",
+    "typescript": "^4.7.2",
+    "glamor": "^2.20.40",
+    "d3-scale": "^2.1.2",
+    "d3-format": "^1.4.5",
+    "lodash": "^4.17.21"
+  },
```

Comment:
I think some extra dependencies snuck in here. Remember to clean them up later.

Reply:
some stuff comes from the styleguide bits i ruthelessly copied into the scrolly repo